src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
```

Comment:
I think I would have another state for currently visibleMovies, just to be clear what's is currently displayed

Reply:
> I think I would have another state for currently visibleMovies, just to be clear what's is currently displayed

Please provide example if you can